### ISSUE TITLE
scope all imports 

### ISSUE THREAD

**Issue by stewartboogert** [g4edge/pyg4ometry#198]
Need to properly scope all imports 

e.g. 

`import numpy as _np`

Makes it much easier to change the import structure in the future

**Comment by gipert** [comment]
I would propose to remove the underscore in front, it's not really needed and makes code prettier :)

**Comment by stewartboogert** [comment]
Agreed but it forms part of what is imported with a module. So if users mechanism is not reading and just "tabbing" around different classes it clutters the available scope
